src/utils/utils.ts

```diff
@@ -150,6 +150,27 @@ export function isES6Set(thing): thing is Set<any> {
     return thing instanceof Set
 }
 
+/**
+ * Returns the following: own keys, prototype keys & own symbol keys.
+ */
+export function getPlainObjectKeys(object) {
+    return unique(
+        ([] as PropertyKey[]).concat.apply(
+            [],
```

Comment:
I think however that in none of the cases it is needed to grab keys from the prototype, so Reflect.keys, and filtering out the enumerable ones, should do the trick (with e.g. propertyIsEnumerable). 

A more accurate name would then be `getOwnEnumerablePropertyNames`.

After a quick inspection of the usage sides, the only function that might run into trouble by not inspecting prototype chain might be `toJS`, it would be good to test that.

Reply:
Cool I'll run with these suggestions, thanks mate! I'll get back with changes as soon as I can.

I did observe in `toJS` tests that we are expecting enumerable properties up the prototype to be output, this initial implementation was to address those failing tests - I can split a test or two out to make this intention a little bit more obvious for us.

Reply:
Pushed up what I think is now the correct implementation, following your advice! Ty! 😄